Dockerfile

```diff
@@ -45,7 +52,14 @@ LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-torc-pi
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
+# Copy artifacts from setup stage
 COPY --from=setup /home/carma/install /opt/carma/install
+
+# Make sure permissions are correct
 RUN sudo chmod -R +x /opt/carma/install
 
-CMD  [ "wait-for-it.sh", "localhost:11311", "--", "roslaunch", "pinpoint", "pinpoint.launch", "remap_ns:=/saxton_cav/drivers" ]
+# Source the ROS setup file
+RUN echo "source /opt/carma/install/setup.bash" >> ~/.bashrc
```

Comment:
install_ros2 gets created starting from autoware.ai and in carma-platform. Since these have both ros1 and ros2 they need to differentiate, but this driver is only ros2 now, so we can just source install/setup.bash